### ISSUE TITLE
batch normalization

### ISSUE THREAD

**Issue by marmus12** [balancap/SSD-Tensorflow#24]
hello, would it be easy to add batch normalization to this model?

**Comment by balancap** [comment]
I think you need to base the SSD detector on the deep network already using batch normalization, like ResNets, Inception or Xception. I did not have time to try out, but it should be possible without too much trouble.
